### PR TITLE
Switch to using ffmpeg

### PR DESCRIPTION
**Pull request by maxnoe**
libav is not maintained anymore and ffmpeg is just *the* standard tool for this

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# remember to add 203x in 10 years
```
Comment (dneise) [inline .gitignore:1]: :+1:

**Comment by dneise** [comment]
you know you are the best, right? Thanks so much for doing this.

**Comment by maxnoe** [comment]
@dneise thx, approve?

**Review by dneise — APPROVED** [review]
works nicely

**Pull request merged**